db/osqlblockproc.c

```diff
@@ -1297,7 +1297,8 @@ static int apply_changes(struct ireq *iq, blocksql_tran_t *tran, void *iq_tran,
         if (out_rc)
             break;
     }
-
+#if 0
```

Comment:
Unless this is comming back on an attr, we could just delete this piece of code?